projects/demo-integrations/cypress/support/commands.ts

```diff
@@ -29,4 +29,21 @@ Cypress.Commands.add('hideHeader', hideHeader);
 Cypress.Commands.add('waitKitDialog', waitKitDialog);
 Cypress.Commands.add('setNightMode', setNightMode);
 
-addMatchImageSnapshotCommand();
+addMatchImageSnapshotCommand({
+    // When fullPage, your application under test
+    // is captured in its entirety from top to bottom.
+    capture: 'fullPage',
```

Comment:
Why do we need fullPage?

Reply:
https://docs.cypress.io/api/commands/screenshot#Arguments
this value is default
 

I just wanted to explicitly value in code to make it easier to read

![image](https://user-images.githubusercontent.com/12021443/152117113-5d4de00b-3563-4b22-8ea8-b4eb61e54fd4.png)